.circleci/config.yml

```diff
@@ -34,7 +34,11 @@ jobs:
 workflows:
   releaser:
     jobs:
+      - hold:
```

Comment:
Thinking about it, this job is an approval, therefore the name should probably indicate what we approve. maybe `approve_release_helm_charts`?

Reply:
yes, that naming makes much more sense. Added